tests/cases/conformance/expressions/optionalChaining/taggedTemplateChain/taggedTemplateChain.ts

```diff
@@ -0,0 +1,4 @@
+declare let a: any;
+a?.`b`;
+
+a?.`b${1}c`;
```

Comment:
In ECMAScript this is an early error to prevent ASI. We introduce a grammar error to reflect the ECMAScript grammar. @DanielRosenwasser can provide more context.